modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -1127,27 +1127,6 @@ public void testAddObjectEntry() throws Exception {
 			).put(
 				"name", listTypeEntry.getName(LocaleUtil.US)
 			).build());
-
-		// Relationship
-
-		String newExternalReferenceCode = RandomTestUtil.randomString();
-
-		_defaultObjectEntryManager.addObjectEntry(
-			dtoConverterContext, _objectDefinition2,
-			new ObjectEntry() {
-				{
-					properties = HashMapBuilder.<String, Object>put(
-						_objectRelationshipERCObjectFieldName,
-						newExternalReferenceCode
-					).build();
-				}
-			},
-			ObjectDefinitionConstants.SCOPE_COMPANY);
-
-		Assert.assertNotNull(
-			_defaultObjectEntryManager.getObjectEntry(
-				_objectDefinition1.getCompanyId(), _simpleDTOConverterContext,
-				newExternalReferenceCode, _objectDefinition1, null));
```

Comment:
@p-albuquerque change the test to assert that the exception will be thrown for now (`AssertUtils.assertFailure`).